src/renderer/fonts/font_options.rs

```diff
@@ -192,35 +192,37 @@ impl FontOptions {
     }
 
     pub fn font_list(&self, style: CoarseStyle) -> Vec<FontDescription> {
+        if style == CoarseStyle::default() {
+            return self.normal.clone();
+        }
```

Comment:
The early return looks ok to me, this change makes sure 1)we respect user settings of normal 2)fallback to normal always applys the corresponding font style